addons/arsenal/stringtable.xml

```diff
@@ -800,6 +806,7 @@
         </Key>
         <Key ID="STR_ACE_Arsenal_exportedClassnameText">
             <English>Exported class name to clipboard</English>
+            <German>Exportiert den Classname in die Zwischenablage</German>
```

Comment:
It's "was exported" not "will export"
"Der Klassenname wurde in die Zwischenablage exportiert"
or "..ablage kopiert"